components/codeflare/codeflare.go

```diff
@@ -71,9 +72,32 @@ func (c *CodeFlare) GetComponentName() string {
 	return ComponentName
 }
 
+func (c *CodeFlare) UpdateStatus(in *status.ComponentsStatus) error {
+	codeFlareStatus, err := c.GetReleaseVersion(in, deploy.DefaultManifestPath, ComponentName)
+
+	if err != nil {
+		in.CodeFlare = &status.CodeFlareStatus{}
```

Comment:
1. in dsc controller the error is not checked, as it throw err or not does not really matter?
2. in the error case, it still is going to set the status for component ? 
```
Name : ""
DsiplayName: ""
Version: 0.0.0
RepoURL: ""
```

Reply:
in case of error or no upstream, just returns empty struct and displays like dashboard: {} and it is handled inside the component itself. So i feel dsc controller doesnt have to handle the error.